crates/fluss/src/client/table/remote_log.rs

```diff
@@ -184,13 +223,34 @@ impl RemoteLogDownloader {
         // opendal::Reader::read accepts a range, so we read in chunks
         const CHUNK_SIZE: u64 = 8 * 1024 * 1024; // 8MB chunks for efficient reading
         let mut offset = 0u64;
+        let mut chunk_count = 0u64;
+        let total_chunks = file_size.div_ceil(CHUNK_SIZE);
 
         while offset < file_size {
             let end = std::cmp::min(offset + CHUNK_SIZE, file_size);
             let range = offset..end;
-
-            // Read chunk from remote storage
-            let chunk = op.read_with(relative_path).range(range.clone()).await?;
+            chunk_count += 1;
+
+            if chunk_count <= 3 || chunk_count % 10 == 0 {
+                eprintln!(
```

Comment:
I'm guessing it's to track the download progress. But why use `eprintln` which is for error print.

Reply:
Good catch. This was supposed to be a debug-level log, but I couldn't find how to print it for a while. I hoped Copliot could help, and it worked! Haha.

Reply:
done